packages/smart-contracts/src/lib/artifacts/ERC20EscrowToPay/index.ts

```diff
@@ -21,6 +21,10 @@ export const erc20EscrowToPayArtifact = new ContractArtifact<ERC20EscrowToPay>(
           address: '0xEbe28A2B7336670Ba752bfEad4a121D2c4FF2464',
           creationBlockNumber: 10461945,
         },
+        goerli: {
```

Comment:
⚠️ there is no contract at that address
https://goerli.etherscan.io/address/0xEbe28A2B7336670Ba752bfEad4a121D2c4FF2464#code